src/main/java/core/basesyntax/MainApp.java

```diff
@@ -12,5 +12,15 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
-
+    public static void main(String[] args) {
+        Bulldozer bulldozer = new Bulldozer();
```

Comment:
What will happen if we write:
Machine bulldozer = new Bulldozer();
and the same with other objects?

Reply:
the result will not change

Reply:
What will happen if we write:
Machine bulldozer = new Bulldozer();
and the same with other objects?

Reply:
How do you think, why do we create abstract class Machine?
And what us the difference using Bulldozer or Machine?

Reply:
In Machine  we add the methods that we need to inplement in child classes.
We can write any Machine subclass object in the Machine class object.